libs/datamodel/core/src/validator/lift.rs

```diff
@@ -148,23 +147,7 @@ impl<'a> LiftAstToDml<'a> {
         field.documentation = ast_field.documentation.clone().map(|comment| comment.text);
         field.arity = self.lift_field_arity(&ast_field.arity);
 
-        if let Some(value) = &ast_field.default_value {
-            let validator = ValueValidator::new(value);
-
-            if let dml::FieldType::Base(base_type, _) = &field_type {
-                match validator.as_default_value_for_scalar_type(*base_type) {
-                    Ok(dv) => field.default_value = Some(dv),
-                    Err(err) => errors.push(err),
-                };
-            } else {
-                errors.push(DatamodelError::new_validation_error(
-                    "Found default value for a non-scalar type.",
-                    validator.span(),
-                ))
-            }
```

Comment:
Is there an equivalent validation for the default attribute?

Reply:
Yes we have something in place in `default.rs`. This error did not even bubble up.